plugins/kapt3/kapt3-base/src/org/jetbrains/kotlin/kapt3/base/javac/KaptJavaLog.kt

```diff
@@ -40,12 +40,14 @@ class KaptJavaLog(
     noticeWriter: PrintWriter,
     override val interceptorData: KaptJavaLogBase.DiagnosticInterceptorData,
     private val mapDiagnosticLocations: Boolean
-) : Log(context, errWriter, warnWriter, noticeWriter), KaptJavaLogBase {
+) : Log(context), KaptJavaLogBase {
     private val stubLineInfo = KaptStubLineInformation()
     private val javacMessages = JavacMessages.instance(context)
 
     init {
-        context.put(Log.outKey, noticeWriter)
```

Comment:
Looking at the git log, this was here because of https://youtrack.jetbrains.com/issue/KT-16146 where KAPT failed if you set `kapt.verbose=true`. That was 5 years ago, so hopefully it isn't needed anymore. However, no test was added in that commit, so if that flag still exists, maybe we can add it to some tests to check that it has no bad effects?

Reply:
The flag is used ins many of the KAPT Gradle integration tests, see: https://github.com/JetBrains/kotlin/blob/master/libraries/tools/kotlin-gradle-plugin-integration-tests/src/test/kotlin/org/jetbrains/kotlin/gradle/Kapt3IT.kt#L52-L54

Reply:
And as far as I understand those are run with multiple JDK versions. That's good enough for me. :)

Everything looks good to me. What do you think @udalov?

Reply:
Yes, looks good.